lldb/source/Host/posix/ConnectionFileDescriptorPosix.cpp

```diff
@@ -730,9 +730,21 @@ ConnectionStatus ConnectionFileDescriptor::ConnectFile(
     struct termios options;
     ::tcgetattr(fd, &options);
 
-    // Set port speed to maximum
+    // Set port speed to the available maximum
+#ifdef B115200
     ::cfsetospeed(&options, B115200);
     ::cfsetispeed(&options, B115200);
+#elif B57600
+    ::cfsetospeed(&options, B57600);
+    ::cfsetispeed(&options, B57600);
+#elif B38400
+    ::cfsetospeed(&options, B38400);
+    ::cfsetispeed(&options, B38400);
+#else
+    if (error_ptr)
+      *error_ptr = Status::FromErrorString("Maximum Baud rate is Unknown");
+    return eConnectionStatusError;
```

Comment:
I should have been clearer, I think this else should be a compiler error:
```
#error Maximum baud rate unknown.
```
This brings the issue to the developer's attention as early as possible.